doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -10,9 +10,58 @@
 
 <a name="6.1"></a>
 ## 6.1 Introduction
+Securing Kubernetes requires several layers of security features to provide end to end security for cloud native applications. It is also important to adopt a layered packaging model which support separation of concerns during image build. A fully integrated security testing should be baked into the CI/CD pipeline. Automated security policies should also be used to flag builds with issue.  Image registries must be monitored to automatically block or replace images with known vulnerabilites, while also ensuring policies are used to gate what can be deployed and who can deploy from the registry.
+
+The following functionalities are recommended for securing kubernetes platform;
+- Image Signing
+- Role-base Access Control
+- Secret Managment
+- Kubernetes Cluster Multi-tenancy security
+  - Tenant can be distinct teams or workload types (Develpoment or Production) within an organisation, each with a namespace
+  - Tenant can also be per dedicated kubernetes cluster
+- Integration with other security ecosystem like monitoring and alerting tools
 
 <a name="6.2"></a>
-## 6.2 Heading
+##  6.2 Principles
+The following are core principles to consider when securing cloud native applications;
+
+- Deploy only secure applications and trusted codes
+- Only deploy applications from validated and verified images
+- Only deploy applications from trusted registries
+- Containers orchestration (Kubernetes) must be secure with administrative boundaries between tenants
+  - Use Namespaces to establish security boundaries between tenants
+  - Create and define cluster network policies
+  - Run a cluster-wide pod security policy
+  - Turn on Audit Logging
+  - Separate sensitive workloads using namespaces
+  - Secure tenant metadata Access
+- Network segmentation using security zoning and network standards must be followed when deploying containers 
+- The Host OS running the containers must be hardened
+- Use container-aware runtime defense tools
+- Enable Role-Based Access Control (RBAC)
 
 <a name="6.3"></a>
-## 6.3 Heading
+## 6.3 Security Perimeters
+When applications or workloads run on Kubernetes, there are several layers which come into picture that govern the security. Each of these layers need to be secured within their perimeters. The various layers that come into picture are:
+
+- Container Registry: A container registry is a repository to manage container images. The access to container registry needs to be secured in order to provide a controlled access.
```

Comment:
Suggest change ".. needs to be secured in order to provide a controlled access" to "... needs to be secured in order to prevent unauthorised access or image tampering."

Reply:
fixed.